### ISSUE TITLE
Idea: show times in budget slider

### ISSUE THREAD

**Issue by david4worx** [PropaneDragon/RushHour#9]
Right now the budget sliders are split in 2 parts, day and night but no clear indication when one is becoming active/inactive.
With this mod day/night budgets are becoming important (finally!) but with no clear indication as to when and what is happening it still feels like walking in absolute darkness.
![Budget slider](http://i.imgur.com/eFrxC1G.png)

Awesome work so far and keep it up!

**Comment by PropaneDragon** [comment]
Good idea. I'll look into how possible this might be and see what I can do.

**Comment by bloodypenguin** [comment]
Isn't sun/moon indicator in the bottom left corner of the screen what you need?

**Comment by david4worx** [comment]
@earalov that is one way I guess. Still, I find it kinda hard to adjust the budget properly (maximize profit) for the night without the actual times. Sometimes I have the feeling adjusting things would screw up my traffic for instance because my observations are off by 2 or 3 hours (either in day or night)

**Comment by PropaneDragon** [comment]
Yeah, I can see what you mean and I'd love to do it later on in development. I want to do it in a way where I won't be messing with too much of the game though, just incase an update gets to it and I have to do a ton of work to get it back up to date.
